docs/tutorials/todo-manager/README.mdx

```diff
@@ -0,0 +1,1097 @@
+
+Once you sign in and back to the MCP inspector, repeat the actions we did in the previous checkpoint to run todo manager tools. This time, you can use these tools with your authenticated user identity. The behavior of the tools will depend on the roles and permissions assigned to your user:
+
+- If you're logged in as a **User** (with only `create:todos` scope):
+  - You can create new todos using the `create-todo` tool
+  - You can only view and delete your own todos
+  - You won't be able to see or delete other users' todos
+
+- If you're logged in as an **Admin** (with all scopes: `create:todos`, `read:todos`, `delete:todos`):
+  - You can create new todos
+  - You can view all todos in the system using the `get-todos` tool
+  - You can delete any todo using the `delete-todo` tool, regardless of who created it
+
+You can test these different permission levels by:
+1. Signing out of the current session (click the "Disconnect" button in MCP inspector)
+2. Signing in with a different user account that has different roles/permissions
+3. Trying the same tools again to observe how the behavior changes based on the user's permissions
+
+This demonstrates how role-based access control (RBAC) works in practice, where different users have different levels of access to the system's functionality.
+
+![MCP inspector todo manager tool result](./assets/result.png)
+
+<Tabs groupId="sdk">
+<TabItem value="python" label="Python">
+
+:::info
+Check out the [MCP Auth Python SDK repository](https://github.com/mcp-auth/python/blob/master/samples/server/todo-manager.py) for the complete code of the MCP server (OIDC version).
+:::
+
+</TabItem>
+<TabItem value="node" label="Node.js">
+
+:::info
+Check out the [MCP Auth Node.js SDK repository](https://github.com/mcp-auth/js/blob/master/packages/sample-servers/src) for the complete code of the MCP server (OIDC version).
+:::
+
+</TabItem>
+</Tabs>
+
+## Closing notes
+
+🎊 Congratulations! You have successfully completed the tutorial. Let's recap what we've done:
+
+- Setting up a basic MCP server with todo management tools (`create-todo`, `get-todos`, `delete-todo`)
+- Implementing role-based access control (RBAC) with different permission levels for users and admins
+- Integrating the MCP server with an authorization server using MCP Auth
+- Configuring the MCP Inspector to authenticate users and use access tokens with scopes to call tools
+
+You may also want to explore some advanced topics, including:
+
+- Using [JWT (JSON Web Token)](https://auth.wiki/jwt) for authentication and authorization
+- Leveraging [resource indicators (RFC 8707)](https://auth-wiki.logto.io/resource-indicator) to specify the resources being accessed
+- Implementing custom access control mechanisms, such as [role-based access control (RBAC)](https://auth.wiki/rbac) or [attribute-based access control (ABAC)](https://auth.wiki/abac)
```

Comment:
i think we've covered these topics in this tutorial?

Reply:
Removed.